tests/Microsoft.Bot.Builder.Expressions.Tests/ExpressionEngineTests.cs

```diff
@@ -397,7 +410,17 @@ public class ExpressionEngineTests
             Test("count(union(createArray('a', 'b'), createArray('b', 'c'), createArray('b', 'd')))", 4),
             Test("count(intersection(createArray('a', 'b')))", 2),
             Test("count(intersection(createArray('a', 'b'), createArray('b', 'c'), createArray('b', 'd')))", 1),
-            
+            Test("skip(createArray('H','e','l','l','0'),2)", new List<object>{"l", "l", "0"}),
+            Test("take(createArray('H','e','l','l','0'),2)", new List<object>{"H", "e"}),
+            Test("subArray(createArray('H','e','l','l','o'),2,5)", new List<object>{"l", "l", "o"}),
+            Test("count(newGuid())", 36),
+            Test("newGuid().indexOf('-')", 8),
+            Test("indexOf(newGuid(), '-')", 8),
+            Test("indexOf(hello, '-')", -1),
+            Test("newGuid().lastIndexOf('-')", 23),
+            Test("lastIndexOf(newGuid(), '-')", 23),
+            Test("lastIndexOf(hello, '-')", -1),
+            Test("newGuid().length()",36),
```

Comment:
although the current implematation support this, this is not expected, so let's don't . to invoke function there